### ISSUE TITLE
Just installed and blocks encounter error

### ISSUE THREAD

**Issue by clobban** [WordPress/gutenberg#9460]
I just installed and enabled the Gutenberg plug-in and (1) it knocked out my TinyMCE Advanced toobars; (2) everytime I tried to add a block I got this error message: "This block has encountered an error and cannot be previewed."  So much for that; disabled it!

**Comment by designsimply** [comment]
Thank you for reporting! This sounds like a plugin conflict. I tried installing and testing adding new blocks to a new page with the TinyMCE Advanced plugin activated but I didn't see the same error. Is https://wordpress.org/plugins/tinymce-advanced/ the plugin you are using?

**Comment by clobban** [comment]
Yes, that’s the one. I had it installed before I added the Gutenberg plugin.
I cannot  figure out blocks in Gutenberg at all. They do not seem to allow any flexibility for layout, e.g., putting a side bar under the page title in the 2017 2-column theme.
I tried dialing TinyMCE Advanced and then installing Gutenberg but it didn’t help.

It’s obviously a very powerful program with a long learning curve. I have time to make an elaborate nearly empty site, or a simple one filled with information. I’ll go with the latter. The themes make it easy to do that, but woe betide anyone who wants to deviate from them!

Chris
